### PR TITLE
Don't prefix used variables with underscore

### PR DESCRIPTION
**Pull request by nmattia**
This replaces a few (used) variables like `_foo` with `foo`.

Some local variables were prefixed with an underscore to differentiate them from a similarly named argument, e.g.:

```js
function foo(bar) {
  const _bar = bar +1;
}
```

Now instead the argument has an underscore appended:

```js
function foo(bar_) {
  const bar = bar_ +1;
}
```

this ensures that by default, `bar` is used.

### CODE REVIEW

File: src/frontend/src/components/authenticateBox.ts [hunk 1]
```diff
@@ -255,22 +255,25 @@ const page = (slot: TemplateResult) => {
...
       // TODO L2-309: do this without reload
       return window.location.reload() as never;
     }
 
-    return _userNumber;
+    return userNumber;
   };
 
-  const _userNumber = userNumber ?? (await askUserNumber());
+  const userNumber = userNumber_ ?? (await askUserNumber());
```
Comment (peterpeterparker) [inline src/frontend/src/components/authenticateBox.ts:276]: technically if you inline the check I think you would maybe not even need `userNumber_` here right?

```
const addRemoteDevice = async (
  connection: Connection,
  userNumber?: bigint
) => {
  // Prompt the user for an anchor (only used if we don't know the anchor already)
  const askUserNumber = async () => {
    const userNumber = await promptUserNumber({
      title: "Add a Trusted Device",
      message: "What’s your Identity Anchor?",
    });
    if (userNumber === "canceled") {
      // TODO L2-309: do this without reload
      return window.location.reload() as never;
    }

    return userNumber;
  };

  return registerTentativeDevice(userNumber ?? (await askUserNumber()), connection);
};
```

(not a call to action or comment, meant as question only)

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/components/authenticateBox.ts (hunk 1 above)
Comment (nmattia) [reply src/frontend/src/components/authenticateBox.ts:276]: yes that's correct! let me give this a think, might be less error prone

Comment (nmattia) [reply src/frontend/src/components/authenticateBox.ts]: @peterpeterparker I've changed it to do like you suggest, thanks! 👍

**Pull request merged**